miro_preprocessor/miro_copy_s3_asset/miro_copy_s3_derivative_asset/src/install_deps.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+pip install -r requirements.txt
```

Comment:
This needs

```bash
set -o errexit
set -o nounset
```

at the top so it can bail out if installing requirements fails.